src/App.tsx

```diff
@@ -345,6 +346,7 @@ function App() {
 
   function login(credential: string | undefined) {
     setIsLoggedIn(true);
+    setOpenAuthDialog(false);
```

Comment:
So now we default to not asking for login, even if auth is enabled?
I think that's fine, but may lead to the odd situation where you send a query and get no replies / errors because you forgot to log in. We should probably address that case soon as well.

Reply:
I believe we should address that together with the APIs. If some API has Auth enabled and gets no ID token in its response, it'll returns an error message that we'll can display to the user using toasts on the query tool.